packages/composite-checkout-wpcom/src/hooks/use-shopping-cart.ts

```diff
@@ -219,6 +228,13 @@ export interface ShoppingCartManager {
 	submitCoupon: ( string ) => void;
 	couponStatus: CouponStatus;
 	couponCode: string | null;
+	updateLocation: ( CartLocation ) => void;
+}
+
+interface CartLocation {
```

Comment:
Along with the earlier suggestion about editing `responseCart` directly, I think this type would fit in nicely in `types/backend/shopping-cart-endpoint`, and once there could shorten the definitions of `RequestCart` and `ResponseCart`.